raspi-config

```diff
@@ -293,6 +293,7 @@ do_configure_keyboard() {
   dpkg-reconfigure keyboard-configuration &&
   printf "Reloading keymap. This may take a short while\n" &&
   invoke-rc.d keyboard-setup start || return $?
+  setsid sh -c 'exec setupcon -k --force <> /dev/tty1 >&0 2>&1'
```

Comment:
Why can't this whole line just be `setupcon -k --force`?

Reply:
I tried it, but I didn't get consistent results.Sometimes the console layout would change immediatelly, sometimes it didn't, especially when doing several changes one after another.

Sending the setupcon command to be executed in tty1 seems to work immediately every time, as that's the expected behaviour of setupcon when executed at the console.

I know back-to-back changes in keyboard layout are probably not a very common scenario, but even so I would think a consistent behaviour is always better, unless there's a reason not to do it this way, of course.